src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,69 @@
-export const Pagination = () => {};
+import React from 'react';
+
+type Props = {
+  total: number // total number of items to paginate
+  perPage: number // number of items per page
+  currentPage: number /* optional with 1 by default */
+  paginate: (page: number) => void
+  previousPage: () => void
+  nextPage: () => void
+};
+
+export const Pagination: React.FC<Props> = (
+  {
+    total, perPage, currentPage, paginate, previousPage, nextPage,
+  },
+) => {
+  const pageNumbers = [];
+
+  for (let i = 1; i <= Math.ceil(total / perPage); i += 1) {
+    pageNumbers.push(i);
+  }
```

Comment:
you can just call getNumbers function instead of this logic

Reply:
in this pageNumbers arr we will collect now all 42 nums but only this amount Math.ceil(total / perPage)

Reply:
```suggestion
   const pageNumbers = getNumbers(1, Math.ceil(total / perPage));
```
it can be something like that with use getNumbers util